packages/homepage/content/articles/codesandbox-ci-embeds/index.md

```diff
@@ -0,0 +1,95 @@
+---
+banner: ./images/oldvsnew.png
+slug: codesandbox-ci-embeds
+authors: ['Ives van Hoorne']
+photo: https://avatars0.githubusercontent.com/u/587016?s=460&v=4
+title: Announcing CodeSandbox CI and all-new Embeds
+description:
+  Today we're pleased to launch CodeSandbox CI - A free continuous integration
+  service purpose-built for open source library maintainers, along with a new
+  look for embeds.
+date: 2019-11-13
+---
+
+## CodeSandbox CI
+
+Maintaining open source projects, while enormously fulfilling, can be
+time-consuming and stressful. After a busy Hacktober, we know from first-hand
+experience what it's like, and we wanted to help library maintainers make the
+best use of their time. Today we're pleased to launch CodeSandbox CI: A free
+continuous integration service purpose-built for open source library
+maintainers.
+
+![](./images/CodeSandboxCI.gif)
+
+## Here's how it works
+
+CodeSandbox CI is a GitHub app that auto-builds your library⁠ from pull
+requests. Whenever someone opens a new pull request, CodeSandbox CI builds a new
+version of your library. Those builds get posted to our registry, so you can
+test it in CodeSandbox or locally, and all without having to publish the build
+to npm⁠. Plus, if the original GitHub issue contains a link to a reproducible
+sandbox, then we'll automatically fork it using the new PR build. That means you
+can review it online with one click.
+
+## Used by the biggest names in OSS
+
+After a successful private beta, CodeSandbox CI is already in use by projects
+like React, Babel, Framer, Material UI, and others. And it's proving to be a
+real time-saver.
+
+**Try it out for yourself by
+[installing the GitHub app](https://github.com/apps/codesandbox)**.
+
+[![Framer tweet](./images/mattgperry_tweet_1181863225756934144.png)](https://twitter.com/mattgperry/status/1181863225756934144?s=20)
```

Comment:
Just use the tweet instead?  
The [`gatsby-remark-embedder`](https://github.com/MichaelDeBoey/gatsby-remark-embedder) plugin (which is installed) will embed this properly.

```suggestion
https://twitter.com/mattgperry/status/1181863225756934144
```

Also delete the image.